pkg/function/restore_data_using_kopia_server.go

```diff
@@ -0,0 +1,261 @@
+package function
+
+import (
+	"context"
+	"fmt"
+
+	crv1alpha1 "github.com/kanisterio/kanister/pkg/apis/cr/v1alpha1"
```

Comment:
Move internal pkg imports to the group below

Reply:
Done !